proposals/dao/cr_oracle_cleanup.ts

```diff
@@ -0,0 +1,162 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc,
+  PcvStats
+} from '@custom-types/types';
+import { getImpersonatedSigner, overwriteChainlinkAggregator, time, expectRevert } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { BigNumber, Contract } from 'ethers';
+
+let pcvStatsBefore: PcvStats;
+let ethPrice8Decimals: string;
+
+const fipNumber = 'cr_oracle_cleanup';
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  // Deploy composite VOLT*1 oracle
+  // This is needed because the VOLT OraclePassthrough does not have a paused() method
+  const compositeOracleFactory = await ethers.getContractFactory('CompositeOracle');
+  const voltOracle = await compositeOracleFactory.deploy(
+    addresses.core,
+    addresses.oneConstantOracle,
+    addresses.voltOraclePassthrough,
+    false
+  );
+  await voltOracle.deployed();
+  logging && console.log(`voltOracle: ${voltOracle.address}`);
+
+  return {
+    voltOracle
+  };
+};
+
+// Do any setup necessary for running the test.
+// This could include setting up Hardhat to impersonate accounts,
+// ensuring contracts have a specific state, etc.
+const setup: SetupUpgradeFunc = async (addresses, oldContracts, contracts, logging) => {
+  // make sure ETH oracle is fresh (for B.AMM not to revert, etc)
+  // Read Chainlink ETHUSD price & override chainlink storage to make it a fresh value
+  ethPrice8Decimals = Math.round((await contracts.chainlinkEthUsdOracleWrapper.read())[0].toString() / 1e10).toString();
+  await overwriteChainlinkAggregator(addresses.chainlinkEthUsdOracle, ethPrice8Decimals, '8');
+
+  // read pcvStats before proposal execution
+  pcvStatsBefore = await contracts.collateralizationOracle.pcvStats();
+
+  // impersonate the rollover signer, and make the Tokemak pool go to next cycle
+  const TOKEMAK_MANAGER_ROLLOVER_ADDRESS = '0x90b6C61B102eA260131aB48377E143D6EB3A9d4B'; // has the rollover role
```

Comment:
Is this Tokemak setup still needed? We withdrew from the pool via a Guardian action